src/Quasar/Advanced/Types.purs

```diff
@@ -146,27 +106,24 @@ parseDir pt =
   <#> (Pt.rootDir </> _)
   # maybe (Left "Incorrect resource") pure
 
+
 type ActionR =
   { operation ∷ Operation
   , resource ∷ Resource
   , accessType ∷ AccessType
   }
 
-newtype Action = Action ActionR
+newtype Action = Action
```

Comment:
This will be fixed one day though! There's an issue open about deriving for constructors using synonyms.